test/games/dread/gui/test_dread_game_export_dialog.py

```diff
@@ -127,9 +138,11 @@ def test_on_input_file_button(skip_qtbot, tmp_path, mocker):
 def test_get_game_export_params(skip_qtbot, tmp_path):
     # Setup
     options = MagicMock()
-    options.options_for_game.return_value.input_path = tmp_path.joinpath("input/game.iso")
-    options.options_for_game.return_value.output_directory = tmp_path.joinpath("output")
-    options.options_for_game.return_value.output_format = "iso"
+    options.options_for_game.return_value = DreadPerGameOptions(
+        cosmetic_patches=DreadCosmeticPatches.default(),
+        input_directory=tmp_path.joinpath("input/game.iso"),
```

Comment:
shouldn't this be a directory, not a file?

Reply:
Yes, but `get_game_export_params` doesn't really care if the input is valid.